### DISCUSSION TITLE
Can't Create Project

### DISCUSSION THREAD

**Discussion by ariefsn** [nuxtrdev/nuxtr-vscode#151] [Questions]
Hi,
I just installed the extension.
When I tried to run the create command, it failed.
Is there any additional step for setup?
![image](https://github.com/user-attachments/assets/84138be5-a6d9-4a1e-8d22-1040e0a5db35)

Thanks

**Comment by adhamfarrag** [comment]
Hi @ariefsn, 

No, there's no additional setup. It's an error and I'm aware of it. Should be fixed with next release. Project creation command works when you use it from a window with an existing project / folder.

**Reply by ariefsn** [reply]
Thanks, it works.
🎉
